core/graph/src/test/java/uk/gov/gchq/gaffer/graph/GraphTest.java

```diff
@@ -489,11 +493,50 @@ public void shouldThrowExceptionIfStoreClassPropertyIsNotSet() throws OperationE
                     .addSchema(new Schema())
                     .storeProperties(new StoreProperties())
                     .build();
+            fail("exception expected");
         } catch (final IllegalArgumentException e) {
             assertEquals("The Store class name was not found in the store properties for key: " + StoreProperties.STORE_CLASS, e.getMessage());
         }
     }
 
+    @Test
```

Comment:
I tend to not use that annotation as it will only check that somewhere in the test method an exception is thrown. Using try/catch allows you to test a precise method throws an exception and to validate the exception has the correct message.